### ISSUE TITLE
Cannot customize access log

### ISSUE THREAD

**Issue by stejoo** [robertdebock/ansible-role-squid#13]
## Describe the bug

PR #11 adds variable `squid_access_log` and mentions one can now customize the location of Squid's log. However this variable is defined in the role vars. Role vars have a higher priority than inventory variables (`group_vars`, `host_vars` or playbook `vars:`). See: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#understanding-variable-precedence

A user is currently unable to redefine the variable via the generally accepted way of using inventory or playbook vars.

## Playbook

Using current `master` checkout of this role.

```yaml
---
# Playbook to configure squid

- name: Configure squid
  hosts: proxy_fwd
  become: true
  vars:
    squid_access_log: /some/other/path/access.log

  roles:
    - role: robertdebock.squid  # at current master branch
```

## Output

When running the playbook with `--diff` the `configure squid` tasks output:
```
+access_log /var/log/squid
```

I would have expected:
```
+access_log /some/other/path/access.log
```

## Environment

- Control node OS: Fedora 36
- Control node Ansible version: 2.13.6 (in a venv)
- Managed node OS: Red Hat Enterprise Linux 8.7 (Ootpa)

**Comment by stejoo** [comment]
Added fix to PR #12

**Comment by robertdebock** [comment]
Thanks for the issue; seems you are completely correct. I'll let CI test your changes and merge on success.
